Models/MuZero_torch_trainer.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, global_agent):
         self.optimizer = self.create_optimizer()
 
     def create_optimizer(self):
-        optimizer = torch.optim.Adam(self.global_agent.parameters(), lr=config.INIT_LEARNING_RATE,
+        optimizer = torch.optim.SGD(self.global_agent.parameters(), lr=config.INIT_LEARNING_RATE,
```

Comment:
We may not need the decayed_learning_rate function if we're using SGD.

Reply:
I've always been a bigger fan of Adam myself. I've found more success with it in the past. Adam is SGD with a few extra tricks thrown into the mix after all